cmake/sdlchecks.cmake

```diff
@@ -1223,3 +1243,10 @@ macro(CheckLibUnwind)
     endif()
   endif()
 endmacro()
+
+macro(CheckNativeMsgBox)
+  set(HAVE_NATIVE_MESSAGEBOXES_H TRUE)
+  if(SDL_WAYLAND)
+    set(HAVE_NATIVE_MESSAGEBOXES_H FALSE)
```

Comment:
The name `HAVE_NATIVE_MESSAGEBOXES_H` is misleading. There is no `native_messageboxes.h`.
I suggest to use `USE_NATIVE_MESSAGEBOXES` instead, or something similar.

Reply:
Fixed, thanks for the suggestion.